### DISCUSSION TITLE
Diary-20-倒一！ | Zi-Zi's Journey

### DISCUSSION THREAD

**Discussion by giscus** [GZ-Metal-Cell/GZ-Metal-Cell.github.io#340] [Announcements]
# Diary-20-倒一！ | Zi-Zi's Journey

2024.07.01-2024.07.11

https://gz-metal-cell.github.io/2024/07/20/Diary-20-%E5%80%92%E4%B8%80%EF%BC%81/

**Comment by promefire** [comment]
排版的神！

**Reply by GZ-Metal-Cell** [reply]
可惜不是论文写作的神😭
